docs/workflows/sdk-setup.mdx

```diff
@@ -204,95 +93,52 @@ platform :ios, '13.0'
-```javascript
-import com.beyondidentity.embedded.sdk.EmbeddedSdk
+<SetupKotlin/>
 
-EmbeddedSdk.init(
-    app: Application,
-    keyguardPrompt: (((allow: Boolean, exception: Exception?) -> Unit) -> Unit)?,
-    logger: (String) -> Unit,
-    biometricAskPrompt: String, /* Optional */
-    allowedDomains: List<String>?, /* Optional */
-)
-```
+Next Steps:
+- [Bind a Passkey](../sdks/kotlin-sdk/overview#binding-a-passkey)
+- [Authentication](../sdks/kotlin-sdk/overview#authentication)
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-android/)
 
 </TabItem>
 <TabItem value="swift" label="Swift">
 
-First, before calling the Embedded functions, make sure to initialize the SDK.
-
-<!-- javascript is used here since swift doesn't highlight at all. -->
-```javascript
-import BeyondIdentityEmbedded
+<SetupSwift/>
 
-Embedded.initialize(
-    allowedDomains: [String] = ["beyondidentity.com"],
-    biometricAskPrompt: String,
-    logger: ((OSLogType, String) -> Void)? = nil,
-    callback: @escaping(Result<Void, BISDKError>) -> Void
-)
-```
+Next Steps:
+- [Bind a Passkey](../sdks/swift-sdk/overview#binding-a-passkey)
+- [Authentication](../sdks/swift-sdk/overview#authentication)
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-swift/)
 
 </TabItem>
 <TabItem value="reactnative" label="React Native">
 
-First, before calling the Embedded functions, make sure to initialize the SDK. This can be done where you register your root component.
-
-```javascript
-import { Embedded } from '@beyondidentity/bi-sdk-react-native';
-
-Embedded.initialize(
-  biometricAskPrompt: string,
-  allowedDomains?: string[]
-): Promise<Success>;
-```
+<SetupReactNative/>
 
-You may also add a listener to log native events with `Embedded.logEventEmitter` after initializing.
-
-#### Usage
-
-```javascript
-import { Embedded } from '@beyondidentity/bi-sdk-react-native';
-
-Embedded.initialize("Please verify it's really you").catch(console.error);
-
-Embedded.logEventEmitter.addListener(
-  'BeyondIdentityLogger',
-  (message: string) => {
-    console.log(message);
-  }
-);
-```
+Next Steps:
+- [Bind a Passkey](../sdks/react-native-sdk/overview#binding-a-passkey)
+- [Authentication](../sdks/react-native-sdk/overview#authentication)
+- [Documentation](https://gobeyondidentity.github.io/bi-sdk-react-native/)
 
 </TabItem>
 <TabItem value="flutter" label="Flutter">
 
-First, before calling the Embedded functions, make sure to initialize the SDK.
-
-<!-- javascript is used here since flutter is not available and dart doesn't highlight at all. -->
-```javascript
-import 'package:bi_sdk_flutter/embeddedsdk.dart';
+<SetupFlutter/>
 
-Embeddedsdk.initialize(
-    String biometricPrompt,
-    bool enableLogging,
-    List<String>? allowedDomains, /* Optional */
-)
-```
+Next Steps:
+- [Bind a Passkey](../sdks/flutter-sdk/overview#binding-a-passkey)
+- [Authentication](../sdks/flutter-sdk/overview#authentication)
```

Comment:
Maybe for now make these todo links so we can link to the other guides when they merge and you can remove the overviews

Reply:
Yeah, I was just using these as placeholders 🙂 I figured since we had existing content, I might as well link to them, until the newer content is merged